src/operationsClientBrowser.ts

```diff
@@ -72,13 +73,13 @@ export {CallSettings, constructSettings, RetryOptions} from './gax';
 
 export {createApiCall} from './createApiCall';
 
-export interface ClientStubOptions {
-  servicePath: string;
-  port: number;
-  // TODO: use sslCreds?: grpc.ChannelCredentials;
-  // tslint:disable-next-line no-any
-  sslCreds?: any;
-}
+// export interface ClientStubOptions {
```

Comment:
Do we need this anymore? Looks like we don't if you are importing it. Might want to delete it.